app/controllers/katello/remote_execution_controller.rb

```diff
@@ -31,13 +31,15 @@ def prepare_composer
 
       def hosts
         host_ids = params[:host_ids].is_a?(String) ? params[:host_ids].split(',') : params[:host_ids]
-        find_bulk_hosts('edit_hosts',
+
+        bulk_params = {
           included: {
             ids: host_ids,
-            search: params[:scoped_search],
-            excluded: params[:excluded]
+            search: params[:scoped_search]
           }
```

Comment:
> In some of bulk host modal controller I've removed code which sets a `search` value on the form that does not exist in the HTML. I've also removed 'scoped_search' from some of the modal views. I think both of them are not needed because those forms work entirely with the _selected_ hosts. Am I right in saying that?

One thing is we could add the excluded hash to the bulk_params outside the included hash:
The bulk_hosts_extenstion looks for this parameter.https://github.com/Katello/katello/blob/05ef323476286317e282ee93a43ba2504e9c09f7/app/controllers/katello/concerns/api/v2/bulk_hosts_extensions.rb#L30. 
I am not sure if people use this outside the UI though.

Reply:
In terms of the remote execution controller it's not used through the UI and also considering that the it's not an API-related controller I decided to remove it since it really had no function that I could determine

Reply:
So from what I understood, we post on /katello/remote_execution route which redirects to this controller but I might be wrong. I think it's safe to drop the excluded param since this has not worked for a long time anyway so nothing outside the modal controller is probably concerned with this.. :+1:

Reply:
Exactly!